### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by raonyguimaraes**
Update to reflect issue https://github.com/Koed00/django-q/issues/216

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ Installation
     $ pip install django-q
 
 
--  Add `django_q` to your `INSTALLED_APPS` in your projects `settings.py`::
+-  Add `django_q` to your `INSTALLED_APPS` in your projects `settings.py` after all other apps::
```
Comment (Eagllus) [inline README.rst]: Why this change?
Nothing indicates that this is true.

I have this running in production

```
INSTALLED_APPS = (
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.humanize',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'widget_tweaks',
    'model_mommy',
    'django_otp',
    'django_otp.plugins.otp_totp',
    'django_otp.plugins.otp_static',

    'main_app',  # Needs to be above django_q to overwride the qcluster config

    'django_q',
    'mailqueue',
    'qrcode',

    'custom_apps',
    'custom_apps',
    'custom_apps',
    'utils'
)
```

**Comment by raonyguimaraes** [comment]
This is what I'm getting if I add django_q before some other apps:

./manage.py migrate
Traceback (most recent call last):
  File "./manage.py", line 19, in <module>
    execute_from_command_line(sys.argv)
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 347, in execute
    django.setup()
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/apps/registry.py", line 89, in populate
    app_config = AppConfig.create(entry)
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django_q/tasks.py", line 11, in <module>
    import cluster
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django_q/cluster.py", line 29, in <module>
    from django_q.models import Task, Success, Schedule
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django_q/models.py", line 16, in <module>
    class Task(models.Model):
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/db/models/base.py", line 100, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/apps/registry.py", line 244, in get_containing_app_config
    self.check_apps_ready()
  File "/projects/mendelmd/venv/lib/python3.5/site-packages/django/apps/registry.py", line 127, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

Here is my INSTALLED_APPS:

INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'django.contrib.sites',
    'django.contrib.humanize',

    #external libs
    'allauth',
    'allauth.account',
    'allauth.socialaccount',
    'crispy_forms',
    'django_select2',

    # 'djcelery',
    'celery',
    'django_celery_results',
    # 'kombu.transport.django',
    # 'django_celery_results',
    # 'django_celery_beat',

    #private apps
    'dashboard',
    'individuals',
    'variants',
    'diseases',
    'genes',
    'pagination',
    'cases',
    'filter_analysis',
    'pathway_analysis',
    'statistics',
    'databases',
    'projects',
    'files',
    'samples',
    'upload',
    'settings',
    'tasks',
    'workers',
    'analyses',
    'formtools',
    'mapps',
]

django_q only works if I add it after my tasks app, which is quite weird.

**Comment by raonyguimaraes** [comment]
Could this be an incompatibility with apps named 'tasks' ?

**Comment by Eagllus** [comment]
Your problem maybe the same as https://github.com/Koed00/django-q/issues/199
(it kinda looks and feels the same) I remembered I started working on a fix for that but never created the pr for it...

**Comment by raonyguimaraes** [comment]
You are right!

Thanks.

_____________________________________________

Raony Guimarães Corrêa Do Carmo Lisboa Cardenas
PhD in Bioinformatics

email: raonyguimaraes@gmail.com
skype/hangouts: raonyguimaraes
phone: +353 83 8389 337
_____________________________________________

**Comment by raonyguimaraes** [comment]
You were right!
